README_Windows.md

```diff
@@ -0,0 +1,47 @@
+# Installation instructions for Windows 10
+These instructions are to install pepys-import for _development_ on Windows 10 - not for deployment on the client's machines. For those instructions see TODO.
+
+1. Install Python 3.7 from www.python.org
+
+2. Create a `pepys` folder, which contains a `libs` folder.
+
+2. Download the 64-bit sqlite3 DLL from https://www.sqlite.org/download.html
+
+3. Unzip that DLL to `libs\sqlite-python`
+
+4. Navigate to the `<python>\DLLs` folder, copy `_sqlite3.pyd` to `libs\sqlite-python`
+
+5. Add the `libs\sqlite-python` folder to your `%PYTHONPATH%` environment variable (create the variable if necessary)
+
+6. Download spatialite from http://www.gaia-gis.it/gaia-sins/windows-bin-NEXTGEN-amd64/mod_spatialite-NG-win-amd64.7z
+
+7. Unzip and put the folder `mod_spatialite-NG-win-amd64` inside the `libs` folder
+
+8. Add that folder to your `%PATH%` variable
+
+9. Install PostreSQL by downloading version 12.2 for Windows x86-64 from https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
+
+10. Go through the installation wizard making sure that you choose to load the 'Stack Builder' application when you exit the wizard.
+
+11. Use the StackBuilder application to install PostGIS, follow the wizard through to the end.
+
+12. Add the Postgres bin directory to the `%PATH%` - eg. `C:\Program Files\PostgreSQL\12\bin\` - but make sure it comes _after_ the `mod_spatialite` folder (hint: using [Rapid Environment Editor](https://www.rapidee.com/en/about) makes it easy to re-arrange entries in the PATH)
+
+12. Clone the [pepys-import](https://github.com/debrief/pepys-import) repository into a folder _inside_ the `pepys` folder
+
+13. Navigate to the `pepys` folder and run `python -m venv venv` to create a virtual environment in a folder called `venv`
```

Comment:
@robintw - should this be the `pepys-import` folder? I guess it must be, since that's where the `requirements.txt` file is that you refer to a little further down.

Reply:
Ah actually it should be the `pepys` folder and I should get the user to `cd` into the `pepys-import` folder later down - I found from experimentation that it helped keep things separate if the virtual environment folder was outside of the `pepys-import` code folder, particularly as we have to recreate it when we install the deployment anyway.
I'll do a small PR to fix this.